Testing/SystemTests/tests/framework/SXDAnalysis.py

```diff
@@ -195,7 +195,7 @@ def runTest(self):
         self.integrated_peaks = sxd.get_peaks(runno, PEAK_TYPE.FOUND, INTEGRATION_TYPE.PROFILE)
 
     def validate(self):
-        intens_over_sigma = [0.0, 17.863, 13.395, 135.109, 0.0]
+        intens_over_sigma = [0.0, 0.0, 27.47, 131.576, 0.0]
```

Comment:
Changed values for the `IntegratePeaksProfile1D` unit tests look good, just wonder what it is about the second peak here that causes the fit to fail? Would it be possible to plot it or fiddle with the I/sigma threshold?

Reply:
Thanks for looking into this, the peak width must have been near the limit then. If you disable this check how does the fit look compared to on main? Cheers!